### ISSUE TITLE
Use a default list of plugins when running "wordup init"

### ISSUE THREAD

**Issue by stefano-b** [wordup-dev/wordup-cli#40]
I think it would be useful to be able to define somewhere a list of plugins to be automatically installed on every run of `wordup init`; usually one would use the same set of basic plugins for every project, and having a preconfigured list to use as a default would save time, instead of having to manually type each of them when prompted.

Ideally the user would be able to set this list in a global config somewhere, ready to be used when starting a new project.

Probably an even better solution would be to have a prompt on the CLI that asks for conformation to use the default set of plugins, maybe even listing them in the console for a quick review, and continuing with the default behavior of asking if one wants to add some more of them manually.

**Comment by shry** [comment]
Very similiar to #38

**Comment by stefano-b** [comment]
Yes, I didn't noticed the other request but it seems to be the same thing
